node/node_test.go

```diff
@@ -1344,6 +1344,14 @@ func testNodeGetESDTsWithRole(t *testing.T, nodeFactory node.NodeFactory, shardI
 	tokenResult, _, err = n.GetESDTsWithRole(testscommon.TestAddressAlice, core.ESDTRoleNFTCreate, api.AccountQueryOptions{}, context.Background())
 	require.NoError(t, err)
 	require.Len(t, tokenResult, 0)
+
+	tokenResult, _, err = n.GetESDTsWithRole(testscommon.TestAddressAlice, "invalid role", api.AccountQueryOptions{}, context.Background())
```

Comment:
The naming for all these tests is a bit unfortunate.
All tests names contain "shouldWork", but they test invalid cases. We should have perhaps separate tests for invalid cases or put inside a `t.Run`